packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 51.5.0 - March 18, 2021
+
+## Added
+
+#### salesforcedx-sobjects-faux-generator
+
+- Define static representation of subset sobjects ([PR #3044](https://github.com/forcedotcom/salesforcedx-vscode/pull/3044))
+
+#### salesforcedx-utils-vscode
+
+- Changes to libraryCommandletExecutor interface to support cancellation ([PR #3054](https://github.com/forcedotcom/salesforcedx-vscode/pull/3054))
+
+## Fixed
+
+#### salesforcedx-sobjects-faux-generator
+
+- Handle undefined results from describe calls ([PR #3064](https://github.com/forcedotcom/salesforcedx-vscode/pull/3064))
+
+#### salesforcedx-vscode-core
+
+- Update deploy and retrieve beta to use 1.1.19 ([PR #3048](https://github.com/forcedotcom/salesforcedx-vscode/pull/3048))
```

Comment:
I think we can keep this generic and add something around the fixes we released in this version of the library. What do you think @sfsholden ?

Reply:
Sounds good. There was a lot that went into that PR though. Here are some of the main highlights that Bryan had me focus on for testing. Are you thinking we add a line for each of these?

1) Fixed timeout issues with deploy retrieve library for operations taking longer than 10 seconds.
2) Fixed output for multi-file components such as CustomObjects and Bundles.
3) Fixed an issue with retrieving static resources.

Reply:
@lcampos @sfsholden Is this update sufficient?

Reply:
@rcoringrato-sfdc yes, I've approved the PR.